function.annouce.validate.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+function validate_ipv6() {
+	// If we get an IPv6 parameter, use that, else, use client's address
+	// The IPv6 address can either be in the form of:
+	// dead:beef::1234 - i.e., raw address, in which case we use port=
+	// -or-
+	// [dead:beef::1234]:12345 - i.e., enbedded port. Unfortunately
```

Comment:
It can also be with brackets and a `port` variable, such as `ipv6 = [dead:beef::1234] port = 12345`